node/src/effect.rs

```diff
@@ -191,7 +192,7 @@ impl<T> Drop for Responder<T> {
     fn drop(&mut self) {
         if self.0.is_some() {
             // This is usually a very serious error, as another component will now be stuck.
-            error!(
+            warn!(
```

Comment:
why are you reducing this to warn ?

Reply:
That was an accident, a result of some debugging, I shall correct it.